src/Command/CommandBase.php

```diff
@@ -857,12 +858,12 @@ protected function doDetermineCloudApplication() {
     }
 
     // Try to guess based on local git url config.
-    if ($cloud_application = $this->inferCloudAppFromLocalGitConfig($acquia_cloud_client)) {
+    if ($prompt && $cloud_application = $this->inferCloudAppFromLocalGitConfig($acquia_cloud_client)) {
```

Comment:
Reading from local Git config doesn't involve a prompt, does it? So is there a reason to skip it if `$prompt == FALSE`?

Reply:
Oh I see, it's because we're overloading this function to "get any app at all". If we decide to keep this approach, maybe there's a better name for the `$prompt` variable.